<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConGUIMainWindow</class>
 <widget class="QMainWindow" name="ConGUIMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>245</width>
    <height>425</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>245</width>
    <height>425</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ConGUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>226</width>
      <height>364</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <widget class="QPushButton" name="xmmeButton">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>10</y>
       <width>80</width>
       <height>28</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>AgentGUI</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="popguiButton">
     <property name="geometry">
      <rect>
       <x>132</x>
       <y>90</y>
       <width>80</width>
       <height>28</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>PopGUI</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="gxparserButton">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>150</y>
       <width>80</width>
       <height>28</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>GXParser</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="simulationButton">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>248</y>
       <width>181</width>
       <height>28</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>ExpGUI</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="visguiButton">
     <property name="geometry">
      <rect>
       <x>83</x>
       <y>328</y>
       <width>60</width>
       <height>28</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>VisGUI</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>245</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSettings"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionCheck_Updates"/>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../congui.qrc">
     <normaloff>:/icons/images/fileopen.png</normaloff>:/icons/images/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Choose Model/Population Files</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../congui.qrc">
     <normaloff>:/icons/images/exit.png</normaloff>:/icons/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="action_About">
   <property name="icon">
    <iconset resource="../congui.qrc">
     <normaloff>:/icons/images/euracelogo.png</normaloff>:/icons/images/euracelogo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="../congui.qrc">
     <normaloff>:/icons/images/ok.png</normaloff>:/icons/images/ok.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionCheck_Updates">
   <property name="icon">
    <iconset resource="../congui.qrc">
     <normaloff>:/icons/images/ok.png</normaloff>:/icons/images/ok.png</iconset>
   </property>
   <property name="text">
    <string>Check Updates</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../congui.qrc"/>
 </resources>
 <connections/>
</ui>
